components/layout/footer.tsx

```
@@ -0,0 +1,19 @@
+import { Github } from "lucide-react";
+
+export function Footer() {
+	return (
+		<div className="bg-linear-to-br/oklch from-slate-50 to-brand-blue/5 border-t border-slate-200">
+			<div className="flex justify-between container mx-auto text-slate-500 py-8 px-6 text-sm">
+				<p>© 2025 PowerForms</p>
+				<a
+					href="https://github.com/austinolig"
+					target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)